spdx-utils/src/main/kotlin/SpdxLicenseStringMapping.kt

```diff
@@ -153,7 +153,8 @@ object SpdxLicenseStringMapping {
             "GNU General Lesser Public License (LGPL) version 2.1" to LGPL_2_1_ONLY.toExpression(),
             "GNU General Public Library" to GPL_3_0_ONLY.toExpression(),
             "GNU General Public License (GPL)" to GPL_3_0_ONLY.toExpression(),
-            "GNU General Public License (GPL), version 2, with the Classpath exception" to GPL_2_0_ONLY.toExpression(),
+            "GNU General Public License (GPL), version 2, with the Classpath exception"
```

Comment:
If this was just wrapped because of the line length / detekt, it should be squash to the commit introducing that line.

Reply:
It had to be wrapped because the value changed. I found that it was wrong while updating the expected test results for this commit.